NEWS.md

```diff
@@ -61,15 +69,23 @@
     missing tics from the server was fixed. There are both client- and
     server- side fixes to fix the problem when playing with older versions
     (thanks MadDog and Mortrixs for help tracking this down).
+  * The macOS launcher now quits automatically when all windows are closed.
+  * The mouse speed calibration thermometer was fixed.
+  * Some bugs were fixed in the protocol used for chocolate-midiproc.exe
```

Comment:
Didn't we remove midiproc just recently for a native Windows MIDI implementation?

Reply:
Seems to be the case. Additionally, outside of the NEWS.md entry for 3.0.0, there's no mention of "midiproc" in the repository.